pkg/deploy/api/types.go

```diff
@@ -388,7 +388,7 @@ type DeploymentLogOptions struct {
 	// Follow if true indicates that the deployment log should be streamed until
 	// the deployment terminates.
 	Follow bool
-	// If true, return previous terminated container logs
+	// If true, return previous deployment logs
```

Comment:
Hm, it's in swagger but I am not sure why the descriptions aren't there

https://github.com/openshift/origin/blob/5597bae3f8ec081a9dacb1d0dc74fdb26f69c95d/api/swagger-spec/oapi-v1.json#L4468